components/NavBar/NavBar.tsx

```diff
@@ -61,22 +39,36 @@ const HeaderContentContainer = styled(Container)`
     margin-right: 5rem;
   }
 `
+const TopBarContainer = styled(TopBar)`
+  width: 100%;
+  padding-bottom: 4rem;  
```

Comment:
Why do we need a padding here, the height are supposed to be 64px according to EDS, now it's 74. Remove this padding 👍  If we need it for something, than something else is wrong

Reply:
yeah the padding is a remnant from the two-row navbar. should just remove it :+1: